mdx/guides/bug-bounty-program.mdx

```diff
@@ -18,22 +18,43 @@ The following are examples of types of vulnerabilities that are of interest:
 -   Unintended contract state
     -   A user is able to update the state of a contract such that it is no longer useable (e.g permanently lock a mutex).
     -   Any assets get unexpectedly "stuck" in a contract with regular use of the contract's public methods.
+    -   An action taken in the staking contracts is applied to an incorrect epoch.
 -   Bypassing time locks
-    -   The `AssetProxyOwner` is allowed to bypass the timelock for transactions where it is not explicitly allowed to do so.
-    -   A user is allowed to bypass the `AssetProxyOwner`.
+    -   The `ZeroExGovernor` is allowed to bypass the timelock for transactions where it is not explicitly allowed to do so.
+    -   A user is allowed to bypass the `ZeroExGovernor`.
+-   Incorrect math calculations
+    -   Overflows or underflow result in unexpected behavior.
+    -   The staking reward payouts are incorrect.
 
 # Scope
 
 The following contracts are in scope of the bug bounty. Please note that any bugs already reported are considered out of scope.
 
+## 3.0
+
+3.0 submissions should be based off of the contracts as of commit [fb8360edfd4f42f2d2b127b95c156eb1b0daa02b](https://github.com/0xProject/0x-monorepo/tree/fb8360edfd4f42f2d2b127b95c156eb1b0daa02b/contracts).
+The following contracts are considered within scope of this bug bounty:
+
+-   [asset-proxy/contracts/src/ERC20BridgeProxy.sol](https://github.com/0xProject/0x-monorepo/blob/fb8360edfd4f42f2d2b127b95c156eb1b0daa02b/contracts/asset-proxy/contracts/src/ERC20BridgeProxy.sol) ([specification](https://github.com/0xProject/0x-protocol-specification/blob/master/asset-proxy/erc20-bridge-proxy.md))
```

Comment:
Nit: Should this contract be included in both the `3.0` and its own bounty?

Reply:
I decided to include it in both just to highlight that it is a new contract.